src/backend/distributed/executor/multi_utility.c

```diff
@@ -395,7 +395,11 @@ multi_ProcessUtility(Node *parsetree,
 	standard_ProcessUtility(parsetree, queryString, context,
 							params, dest, completionTag);
 
-	PostProcessUtility(parsetree);
+	/* don't run post-process code for local commands */
+	if (ddlJobs != NIL)
+	{
+		PostProcessUtility(parsetree);
```

Comment:
Is there any reason not to move this into the `if (ddlJobs != NIL)` block below?